Makefile

```diff
@@ -13,6 +13,7 @@ export PATH := $(path_to_add):$(PATH)
 
 GO        := GO111MODULE=on go
 GOBUILD   := CGO_ENABLED=1 $(GO) build $(BUILD_FLAG) -trimpath
+GOBUILDCOVERAGE := GOPATH=$(GOPATH) CGO_ENABLED=1 cd tidb-server; $(GO) test -coverpkg="../..." -c .
```

Comment:
Why enable CGO?

Reply:
`GOBUILDCOVERAGE` generates a binary that serves the same way with `tidb-server`, this is to make the build as close to `GOBUILD` as possible.

Reply:
We do not need CGO any more, we should remove it after the PR merged.

Reply:
Sure~